### ISSUE TITLE
Не отображаются маршруты

### ISSUE THREAD

**Issue by danskay** [mavrikkk/ha_routes#7]
Вроде бы все сделано по инструкции, но маршруты не отображаются. 
Виртуальные сенсоры создаются и получают данные. 

<img width="376" alt="Screenshot 2021-10-28 at 11 31 45" src="https://user-images.githubusercontent.com/92254904/139200663-ca5f3d5d-8f19-46f3-940a-c8445d0a5b3c.png">

Но на карте всегда Москва)
Мой конфиг:

route:
haddr: https://xx.xxxxx.ru
days: 3
mindst: 0.5
time_zone: +5
token: (тут постоянный токен)
devices:
       - device_tracker.telefon1
       - device_tracker.telefon2

 
haddr - это внешний адрес HA
Home Assistant 2021.10.6

Что я делаю не так?)

**Comment by DIMMonchik** [comment]
Точки на карте отображаются только когда подключаешься по внешнему адресу, а по локальному нет. В чем может быть причина и можно ли это исправить?

**Comment by drey1k** [comment]
Не совсем удобная и законченная интеграция на мой взгляд.
Кроме того, в стандартном элементе "Карта" HA UI [https://www.home-assistant.io/lovelace/map] имеется возможность отображения истории положений трекера. Для этого необходимо в конфигурацию элемента добавить параметр hours_to_show, который определяет количество часов для загрузки истории. В моём случае эта история полностью совпадает с историей местоположений виртуального трекера интеграции "route".

**Comment by mavrikkk** [comment]
Да, не совсем законченная, я бы даже сказал заброшенная ) заброшенная как раз по причине появления истории в самом HA. Когда данная интеграция создавалась, в HA вообще нельзя было показывать маршруты.
Кроме того, у данной интеграции до сих пор остались некоторые преимущества (правда, я не знаю, работает ли она в современных версиях ибо сам не пользуюсь ввиду отсутствия объекта слежения)...в частности тут есть стрелочки с направлением и порядковые номера точек с указанием времени в этой точке, что удобнее при их большом количестве. Кроме того, в родной интеграции не посмотреть статистику за "сегодня"...можно лишь Н-часов назад. Да можно выставить 24 часа, но это будет совпадать с "сегодня" лишь в 0 00 часов )

**Comment by esnet146** [comment]
Так надо бы починить, полезная вещь! именно по причине недостаточности штатной.  :-)
Пока все как и у автора ишью

**Comment by uvaxut** [comment]
Почему-то friendly_name не отображается на карте, вместо этого отображается device_id. И точек как-то мало: на расстоянии 10 км отображается 3 точки, хотя mindst=0.01
